botbot/apps/plugins/core/logger.py

```diff
@@ -10,6 +12,12 @@ class Config(config.BaseConfig):
         help_text="Don't log lines starting with this string"
     )
 
+def should_ignore_text(text, ignore_prefix):
+    return any(
+        (re.match(prefix, text, flags=re.IGNORECASE) is not None)
```

Comment:
> since the rate at which chat messages are coming in is so low

If we want to (try and) push this to upstream, we can't base performance considerations on only our own use case.